crates/holochain_types/src/dna/dna_bundle.rs

```diff
@@ -152,8 +154,8 @@ impl DnaBundle {
                         .or_else(|| manifest.integrity.network_seed.clone())
                         .unwrap_or_default();
 
-                    dna_def.network_seed = network_seed;
-                    dna_def.properties = properties;
+                    dna_def.phenotype.network_seed = network_seed;
+                    dna_def.phenotype.properties = properties;
```

Comment:
not your responsibility, but here's another place where `origin_time` was not included as a parameter in the original function, and hence is neglected to be updated here. This would be another use case for `DnaPhenotypeOption`.

Reply:
It unravels a bigger refactoring which I ask you to include in your follow up PR.

Here's my suggestion for `dna_bundle#to_dna_def`:
```rust
/// Convert to a DnaDef
pub fn to_dna_def(
    &self,
    integrity_zomes: IntegrityZomes,
    coordinator_zomes: CoordinatorZomes,
    phenotype: DnaPhenotypeOption,
) -> DnaResult<(DnaDefHashed, DnaHash)> {
    match &self.manifest().0 {
        DnaManifest::V1(manifest) => {
            let mut dna_def = DnaDef {
                name: manifest.name.clone(),
                phenotype: DnaPhenotype {
                    network_seed: manifest.integrity.network_seed.clone().unwrap_or_default(),
                    properties: SerializedBytes::try_from(
                        manifest.integrity.properties.clone().unwrap_or_default(),
                    )?,
                    origin_time: manifest.integrity.origin_time.into(),
                },
                integrity_zomes,
                coordinator_zomes,
            };

            // If no phenotype overrides, then the original hash is the same as the current hash
            let original_hash = DnaHash::with_data_sync(&dna_def);
            // Otherwise, record the original hash first, for version comparisons.
            if phenotype.has_some_option_set() {
                dna_def.phenotype.network_seed = phenotype
                    .network_seed
                    .unwrap_or_else(|| dna_def.phenotype.network_seed);
                dna_def.phenotype.properties = phenotype
                    .properties
                    .unwrap_or_else(|| dna_def.phenotype.properties);
                dna_def.phenotype.origin_time = phenotype
                    .origin_time
                    .unwrap_or_else(|| dna_def.phenotype.origin_time);
            }
            Ok((DnaDefHashed::from_content_sync(dna_def), original_hash))
        }
    }
}
```